### DISCUSSION TITLE
Schmid Factor over Pole Figure in uniaxial tension

### DISCUSSION THREAD

**Discussion by maxencebuttard** [mtex-toolbox/mtex#2197] [Ask Anything] [answered]
Hello everyone,

I am currently trying to plot Schmid Factor for a specific stress condition `sigma = -stressTensor.uniaxial(vector3d.Y)` over a pole figure. I know that it is possible to plot a Schmid Factor spherical function for different tension direction but I would prefer to plot a Schmid Factor for different orientations for a specific stress condition.

For now here is my code 
```matlab
% Define first slip system
n1 = Miller(1,1,1,ebsd('Cubic').CS,'hkl');
d1 = Miller(1,-1,0,ebsd('Cubic').CS,'uvw');

sS1 = slipSystem(d1,n1);
sSAll1 = sS1.symmetrise('antipodal');
sSLocal = ebsd('Cubic').orientations*sSAll1;

% Calculate Schmid factors for each orientation
SF= sSLocal .SchmidFactor(sigma);
[SFMax1, id1] = max(abs(SF), [], 2);
```

But I don't know how to plot it superimposed to a 100 pole figure.

**Comment by ralfHielscher** [comment, marked as answer]
How about
```matlab
plotPDF(ebsd('Cubic').orientations, SFMax1, Miller(1,0,0,ebsd('Cubic').CS))
```
Ralf.

**Reply by maxencebuttard** [reply]
Dear Ralf,

Thanks a lot for your answer. I am now trying to show on a PDF the evolution of the Schmid Factor over a pole figure for a dedicated sigma tensor. I am trying : ori represent 5k different orientations.
`plotPDF(ori, SFMax, Miller(1,1,0,ori.CS),'antipodal','smooth');colorbar` 
However for memory reasons the plot is not possible. Do you have any idea for using maybe spherical function to go faster ?

Bests
